include/solver.h

```diff
@@ -229,6 +229,15 @@ class AbsSmtSolver
    */
   virtual Term make_symbol(const std::string name, const Sort & sort) = 0;
 
+  /** Look up a symbol by name.
+   *  If no symbol of that name has been declared, throws
+   * IncorrectUsageException
+   *
+   *  @param name the name of the symbol to look up
+   *  @return the Term representation of the corresponding symbol
+   */
```

Comment:
I recommend expanding here, maybe with some of the material in the PR description.

Reply:
good idea! added with b4df4d3cf1a07327a7b52b3cf5ce98494ebd3223